src/main/features/dashboard/views/search.njk

```diff
@@ -0,0 +1,30 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, errorSummary, searchInput %}
+
+{% set title = t('Money claim') %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <form novalidate method="post">
+        {{ csrfProtection(csrf) }}
+
+        <h1 class="heading-large">{{ t('Manage your claims') }}</h1>
+
+        <h2 class="heading-medium form-group-compound">Find a claim</h2>
+        <p>Once you've found a claim you can:</p>
+
+        <ul class="list-bullet form-group">
+          <li>confirm you've served the claim form on the defendant</li>
```

Comment:
missing translation throughout this page

Reply:
done